pom.xml

```diff
@@ -243,6 +244,13 @@
     <build>
         <defaultGoal>install</defaultGoal>
 
+        <testResources>
+            <testResource>
+                <filtering>true</filtering>
+                <directory>${basedir}/src/test/projects</directory>
```

Comment:
resources are accessible on the test's classloader, having them in a separate folder maintains them isolated